homeassistant/components/clickatell/notify.py

```diff
@@ -20,20 +25,24 @@
 )
 
 
-def get_service(hass, config, discovery_info=None):
+def get_service(
+    hass: HomeAssistant,
+    config: ConfigType,
+    discovery_info: DiscoveryInfoType | None = None,
+) -> BaseNotificationService:
```

Comment:
Please use more precise typing here when possible.
```suggestion
) -> ClickatellNotificationService:
```